pkg/certificate/cleanup_test.go

```diff
@@ -283,7 +312,7 @@ var _ = Describe("CABundle cleanup", func() {
 				},
 				{
 					notBefore: -34 * time.Hour,
-					notAfter:  -33 * time.Hour,
+					notAfter:  90 * time.Hour,
 				},
```

Comment:
what happens if all certs are past the cleaning time? should we remove all or keep at least one (if it's still valid of course)?

Reply:
Going to add a test for that, could happend that pod is down beyond the overlap duration and when it goes up it will clean al CABundle and that's too bad, it should keep the one with older issued date.